recipes/msys2/all/conanfile.py

```diff
@@ -53,11 +46,13 @@ class MSYS2Conan(ConanFile):
         "exclude_files": ["ANY"],
         "packages": ["ANY"],
         "additional_packages": [None, "ANY"],
+        "no_kill": [True, False]
     }
     default_options = {
         "exclude_files": "*/link.exe",
         "packages": "base-devel,binutils,gcc",
         "additional_packages": None,
+        "no_kill": False,
```

Comment:
this option should be removed from package id

Reply:
Oops! I for this was not resolved 🙈

Reply:
> this option should be removed from package id

Hi @SpaceIm,

Good catch. Thanks